dev-proxy-plugins/Inspection/DevToolsPlugin.cs

```diff
@@ -61,32 +68,67 @@ private static int GetFreePort()
 
     private string GetChromiumProcessName(DevToolsPluginConfiguration configuration)
     {
+        if (!string.IsNullOrWhiteSpace(configuration.PreferredBrowserPath))
+        {
+            Logger.LogInformation("PreferredBrowserPath was set to {path}. Ignoring PreferredBrowser setting.", configuration.PreferredBrowserPath);
+            return configuration.PreferredBrowserPath;
+        }
+
+        // installation steps for browsers: https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps#install-google-chrome-for-linux
+
         return configuration.PreferredBrowser switch
         {
-            PreferredBrowser.Chrome => RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
-                ? Environment.ExpandEnvironmentVariables(@"%ProgramFiles%\Google\Chrome\Application\chrome.exe")
-                : "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome",
-            PreferredBrowser.EdgeDev => RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
-                ? Environment.ExpandEnvironmentVariables(@"%ProgramFiles(x86)%\Microsoft\Edge Dev\Application\msedge.exe")
-                : "/Applications/Microsoft Edge Dev.app/Contents/MacOS/Microsoft Edge Dev",
-            _ => RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
-                ? Environment.ExpandEnvironmentVariables(@"%ProgramFiles(x86)%\Microsoft\Edge\Application\msedge.exe")
-                : "/Applications/Microsoft Edge.app/Contents/MacOS/Microsoft Edge"
+            PreferredBrowser.Chrome => RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? Environment.ExpandEnvironmentVariables(@"%ProgramFiles%\Google\Chrome\Application\chrome.exe")
+                : RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
+                : RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? "/usr/bin/google-chrome"
+                : throw new NotSupportedException("Unsupported operating system."),
+            PreferredBrowser.EdgeDev => RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? Environment.ExpandEnvironmentVariables(@"%ProgramFiles(x86)%\Microsoft\Edge Dev\Application\msedge.exe")
+                : RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? "/Applications/Microsoft Edge Dev.app/Contents/MacOS/Microsoft Edge Dev"
+                : RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? "/usr/bin/microsoft-edge-dev"
```

Comment:
Seems like we lost Edge

Reply:
that's right,  sorry for that. GH doesn't really show those changes well.